.travis.yml

```diff
@@ -37,6 +36,27 @@ script:
   - misspell -error -locale US $GOFILES
   - gocyclo -over 50 $GOFILES
   - golint -set_exit_status $GOFILES
-  - staticcheck ./cmd/*/*.go ./pkg/*/*.go *.go
+  - staticcheck ./cmd/*/*.go *.go # TODO(brooke): later add ./pkg/*/*.go
   - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco install -y mingw; export PATH=/c/tools/mingw64/bin:"$PATH";fi
 after_success:
+  - bash <(curl -s https://codecov.io/bash)
+  - make
+  - make client
+  - make docker
+before_deploy:
+  - make dist
+after_deploy:
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  - make release-push
+deploy:
+  provider: releases
+  api_key:
+    secure: Ai7efi3yilhxCCjCW6QSQrkk0aYF6unjBDdVywotXg1jTzrNnL1KFPR7vK5AHcU9iakhSrvRYY/wYHs9MV/al+FmTEBHD5QB5+V7QdqiOVVCDjbItP3M16ni14q5s2MwfPsgELIAJoKGIgjdJ7Rwbtd04Oqa1GmA4BzVkr+0TIbbRO85ds7jZUUnpQY3T0MGT8sPoLr4K/iBj6M9BLDlT6j9sEWjVscFkNlN5Q9q7M3/QyZvRlASvXg6FaBhDud1Sn0AOAOJLu0Nq4rv3ZK9+PeURovNalFcq20EwHyoan9IacZ73nPWerwov2JMWBCVcdGffmkZ2DlUa/T15kQkrflPn48CSn1hq5jut9yZpNg7a+BTEOSdlzSb78aS10JRjQ5Ke0Y/6xc5bIeKzgWeICpSGkl1A3rQ6DN8ZRL3gIfx4FfaBDoszQcZQFrmMm21g/QdN1m+MxP9U5bEODWdl9PAo27n11UY5ItMymHw5qaGw3HXUbHk7X+qUHjEWtW/z4dQQJMAzfXCWrx92tEavFE9/gvYtybpn1dfw1O2FlxCxSCkk2AejC/E6RMCgGaDV3SvIzYn+ai0ijwn8o4/fKoaKc8vjIpCDXGvWxAfBkhFRkQ38n3fpBhgYp8THD98jU3XNXsN0ltFfCEJQ3KpPGyCUbbvBnsqPvols0HuaX4=
```

Comment:
Did you copy this from another repo? We setup this with `travis setup releases` (ruby gem, I think)

Reply:
I did so should I remove

Reply:
Yeah so we can fix when needed!